page-objects/modals/createSpaceModal.ts

```diff
@@ -30,4 +32,21 @@ export class CreateSpaceModal {
     await this.button.waitFor();
     await this.button.click();
   }
+
+  async assertNameInputHasError() {
+    await this.nameInput.waitFor();
+    await expect(this.nameInput).toHaveClass(/error/);
+    await expect(this.nameInput).toHaveCSS("border-color", "rgb(211, 61, 68)");
```

Comment:
To działa jak soft asercje? Jeśli nie to trzeba rozbić na pojedyncze.

Reply:
Czemu? Nie jest to soft asercja, ale imo nie ma sensu rozbijać tego na dwie różne asercje - w teście musiało by być coś w stylu `assertInputHasClass(...)` czy `assertErrorMessageHasRedColor()`, co imo nie jest czytelne. Metoda ta teraz sprawdza, czy input "jest w stanie errora", a co to dokładnie oznacza, to sprawa techniczna